demisto_sdk/commands/create_artifacts/content_creator.py

```diff
@@ -148,17 +224,17 @@ def add_tools_to_bundle(tools_dir_path, bundle):
                         zipf.write(os.path.join(root, file_name), file_name)
                 zipf.close()
 
-    @staticmethod
-    def copy_playbook_yml(path, out_path):
+    def copy_playbook_yml(self, path, out_path):
         """
         Add "playbook-" prefix to playbook file's copy destination filename if it wasn't already present
         """
         dest_dir_path = os.path.dirname(out_path)
         dest_file_name = os.path.basename(out_path)
         if not dest_file_name.startswith('playbook-') and tools.find_type(path) == 'playbook':
             new_name = '{}{}'.format('playbook-', dest_file_name)
-            out_path = os.path.join(dest_dir_path, new_name)
+            out_path = self.add_suffix_to_file_path(os.path.join(dest_dir_path, new_name))
```

Comment:
why do we need the suffix

Reply:
as discussed we wanted to add the option to add a suffix to the file path in order to distinguish between the files once we merge 2 zip files (one from regular release branch and the other from 4.1 branch)